mtags/src/main/scala/scala/meta/internal/pc/ScriptFirstImportPosition.scala

```diff
@@ -27,10 +27,14 @@ import scala.meta._
 object ScriptFirstImportPosition {
 
   val usingDirectives: List[String] = List("// using", "//> using")
+  val shebang = "#!"
   val ammHeaders: List[String] = List("// scala", "// ammonite")
 
+  private def adjustShebang(text: String): String =
+    text.replaceFirst(shebang, s"//$shebang")
```

Comment:
Potentially, you could have `"#!"` somewhere in the code. We should just replace the first line. I think this is already being done in Trees

Reply:
I don't think it's that easy, in `Trees` it is in first line, but here it is after script wrapper. If `#!` is somewhere in code, we would not get there with `skipPrefixesOffset`, so it doesn't change anything. The only thing is, if we have `using` directives first and then shebang, then the import is inserted after it, but I think thats correct behaviour